stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/handler/NotificationConfig.kt

```diff
@@ -36,4 +36,6 @@ public data class NotificationConfig(
      * Allows enabling/disabling showing notification after receiving a push message.
      */
     val shouldShowNotificationOnPush: () -> Boolean = { true },
+
+    val requestPermissionOnAppLaunch: () -> Boolean = { true },
```

Comment:
This config need to details what it will do internally on the SDK.
What happens if I leave it as true?
What happens if I put it as false?
How should I request permission on the case it is false?

Reply:
Basically, if a customer keeps it as `true` - then no extra work is required from the customer side, because we will handle permissions request on the SDK side

If the customer makes this value `false` then it is a customer's responsibility to request notification permission within their app. Maybe they already do it much earlier in their app, how knows 🤷‍♂️